apps/src/code-studio/pd/application_dashboard/quick_view_table.jsx

```diff
@@ -95,8 +96,12 @@ export default class QuickViewTable extends React.Component {
     this.context.router.push(`/${this.props.path}/${id}`);
   };
 
+  constructRows() {
+    return this.props.statusFilter ? this.props.data.filter(row => (row.status === this.props.statusFilter)) : this.props.data;
```

Comment:
nit: don't need the inner parens: `filter(row => row.status === this.props.statusFilter)`